src/ziggurat/messaging/dead_set.clj

```diff
@@ -1,22 +1,64 @@
 (ns ziggurat.messaging.dead-set
-  (:require [ziggurat.messaging.consumer :as consumer]
-            [ziggurat.messaging.producer :as producer]))
+  (:require [langohr.basic :as lb]
+            [langohr.channel :as lch]
+            [sentry-clj.async :as sentry]
+            [ziggurat.config :refer [get-in-config]]
+            [ziggurat.sentry :refer [sentry-reporter]]
+            [ziggurat.messaging.connection :refer [connection]]
+            [ziggurat.messaging.consumer :as consumer]
+            [ziggurat.messaging.producer :as producer]
+            [ziggurat.messaging.util :refer :all]))
+
+(defn- try-consuming-dead-set-messages [ch ack? queue-name topic-entity]
+  (try
+    (let [[meta payload] (lb/get ch queue-name false)]
```

Comment:
I wish to keep the rabbitMQ logic encapsulated in consumer namespace, what if we keep this function in consumer namespace itself or abstract it out some other way?